### PR TITLE
Issue #DP-554 Fix: Project name is displayed on top in the CSV

### PR DESCRIPTION
**Pull request by niteshkesarkar**

### CODE REVIEW

File: src/app/client/src/app/modules/core/services/programs/programs.service.ts [hunk 1]
```diff
@@ -883,15 +883,14 @@ export class ProgramsService extends DataService implements CanActivate {
       }));
   }
 
-  downloadReport(filename, title, headers, tableData) {
+  downloadReport(filename, headers, tableData) {
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/core/services/programs/programs.service.ts:886]: @niteshkesarkar What is I want to pass the title?

Comment (niteshkesarkar) [reply src/app/client/src/app/modules/core/services/programs/programs.service.ts]: @twsvaishali - Updated the code to accept the config and merge those configs with default confiig. So we will be able to customize it as per our need.

File: src/app/client/src/app/modules/core/services/programs/programs.service.ts [hunk 2]
```diff
@@ -883,20 +883,22 @@ export class ProgramsService extends DataService implements CanActivate {
       }));
   }
 
-  downloadReport(filename, title, headers, tableData) {
-    const options = {
-      filename: filename,
+  downloadReport(csvDownloadConfig) {
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/core/services/programs/programs.service.ts:886]: this can simply be config

Comment (niteshkesarkar) [reply src/app/client/src/app/modules/core/services/programs/programs.service.ts]: @twsvaishali - renamed to config

**Pull request merged**